webui/react/src/e2e/models/pages/WorkspaceList.ts

```diff
@@ -36,7 +52,18 @@ export class WorkspaceList extends DeterminedPage {
     parent: this,
     selector: '[data-testid="newWorkspace"]',
   });
-  // TODO missing grid list
+  readonly gridListRadioGroup = new GridListRadioGroup({
+    parent: this,
+    selector: '[data-testid="grid-list-radio-group"]',
+  });
+  readonly table = new InteractiveTable({
+    parent: this,
+    tableArgs: {
+      attachment: '[data-testid="table"]',
```

Comment:
just fyi, this has `  readonly defaultSelector = 'div[data-test-component="interactiveTable"]';`. if i had used data-testid instead, then it would've conflicted here. that's why i like to use data-test-component, in case a conflict might arise.